xbmc/ApplicationStackHelper.cpp

```diff
@@ -283,3 +283,15 @@ CApplicationStackHelper::StackPartInformationPtr CApplicationStackHelper::GetSta
   }
   return m_stackmap[key];
 }
+
+CApplicationStackHelper::StackPartInformationPtr CApplicationStackHelper::GetStackPartInformation(
+    const std::string& key) const
+{
+  const auto it = m_stackmap.find(key);
+  if (it == m_stackmap.end())
+  {
+    StackPartInformationPtr value(new StackPartInformation());
+    return value;
+  }
+  return m_stackmap.find(key)->second;
```

Comment:
why another `find`? would `return it->second`work?

Reply:
no reason just stupid